.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  check-code-style:
+    name: Code Style
+    uses: playframework/.github/.github/workflows/cmd.yml@v2
+    with:
+      cmd: sbt validateCode
+
+  check-binary-compatibility:
+    name: Binary Compatibility
+    uses: playframework/.github/.github/workflows/binary-check.yml@v2
+
+  check-docs:
+    name: Docs
+    uses: playframework/.github/.github/workflows/cmd.yml@v2
+    with:
+      cmd: sbt doc
+
+  test:
+    runs-on: macos-latest
```

Comment:
@mkurz WDYT? I think we don't need a Mac OS for check compatibility with Safari. We can use Ubuntu from our reusable workflows and check all other browsers except Safari 😄

Reply:
Probably yes, I am looking into this whole thing right now, give me a bit time.